src/MatchingResult.php

```diff
@@ -9,14 +9,14 @@
 
 final class MatchingResult implements MiddlewareInterface
 {
-    private $success;
+    private ?bool $success;
 
     /**
      * @var Route
      */
-    private $route;
-    private $parameters = [];
-    private $methods;
+    private Route $route;
+    private array $parameters = [];
+    private array $methods;
```

Comment:
Is there a reason not to initialize it?

Reply:
Is it a good idea to initialize it with `Method::ANY`?

Reply:
Yes, that's fine.